File: OPENAPI-REST-API/swagger-client/dart/lib/model/service_categories.dart
part of swagger.api;

class ServiceCategories {
  
  ServiceCategories();

  @override
  String toString() {
    return 'ServiceCategories[]';
  }

  ServiceCategories.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
  }

  Map<String, dynamic> toJson() {
    return {
     };
  }

  static List<ServiceCategories> listFromJson(List<dynamic> json) {
    return json == null ? new List<ServiceCategories>() : json.map((value) => new ServiceCategories.fromJson(value)).toList();
  }

  static Map<String, ServiceCategories> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, ServiceCategories>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new ServiceCategories.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/openapi-client/cpp-tiny/lib/Models/ServiceCategories.cpp


#include "ServiceCategories.h"

using namespace Tiny;

ServiceCategories::ServiceCategories()
{
}

ServiceCategories::ServiceCategories(std::string jsonString)
{
	this->fromJson(jsonString);
}

ServiceCategories::~ServiceCategories()
{

}

void
ServiceCategories::fromJson(std::string jsonObj)
{
    bourne::json object = bourne::json::parse(jsonObj);


}

bourne::json
ServiceCategories::toJson()
{
    bourne::json object = bourne::json::object();


    return object;

}



